getting-started/gopher-run/cmd/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the offical Golang image to create a build artifact.
```

Comment:
Please move `Dockerfile`, `go.mod`, and `go.sum` to the root directory. Then update the module path in `go.mod` to match this repository (`github.com/GoogleCloudPlatform/...`).

Reply:
Done